### ISSUE TITLE
<swaggerInternalFilter> is not working when <supportSpringMvc> is set to true

### ISSUE THREAD

**Issue by deramasa** [kongchen/swagger-maven-plugin#118]

**Comment by kongchen** [comment]
You're right, springmvc mode doesn't not support `swaggerInternalFilter` now.

**Comment by deramasa** [comment]
Is there any specific reason why swaggerInternalFilter is not supported for springmvc? Please let me know by when it will be available.

**Comment by kongchen** [comment]
The reason is I've no enough spare time to implement that, pull request is welcomed.
